amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/optimizing/OptimizingQueue.java

```diff
@@ -468,6 +468,7 @@ public void acceptResult(TaskRuntime taskRuntime) {
         }
       } catch (Exception e) {
         LOG.error("accept result error:", e);
+        throw e;
```

Comment:
I agree that we should throw the exception rather than just printing logs.
But we may just remove the catch blocks because we will print error logs in the service proxy.